.github/workflows/build.yaml

```diff
@@ -1,16 +1,26 @@
 name: Build
 
 on:
+  push:
+    paths-ignore:
+      - "docs/**"
+      - "logo/**"
+      - "examples/**"
+      - "**.md"
+    branches: 
+      - "master"
+
   pull_request:
-    branches: [ master ]
+    branches: 
+      - "master"
 
 jobs:
-  build_and_push:
+  build_job:
     name: Build
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Nit: Let's keep the released version.

Reply:
Released verion is 20-latest, that equal to latest

Reply:
I was referring to use a released version but it probably does not matter in this case.